src/lib.rs

```diff
@@ -186,6 +190,95 @@ pub fn find_configuration_file() -> String {
     }
 }
 
+/// A wrapper around `encrypt_single_file` that takes strings instead of paths.
+///
+/// This makes it easier to call from command line arguments.
+pub fn encrypt_single_file_path(
+    input_file: &str,
+    output_file: Option<String>,
+    encryption_key_string: Option<String>,
+) {
+    let input_file_path = Path::new(input_file).to_path_buf();
```

Comment:
It looks like I missed this in my initial review. Since [PathBuf](https://doc.rust-lang.org/std/path/struct.PathBuf.html) implements `From` & `Into` traits for strings, we can do `PathBuf::from(input_file)` instead. This approach seems to be the preferred one in [the documentation](https://doc.rust-lang.org/std/path/struct.PathBuf.html).

With `Into` trait, we don't even need the extra `input_file_path` variable, we can just pass it into `encrypt_single_file` like so:

```
encrypt_single_file(input_file.into(), output_file_path, encryption_key_string)
```

---

`Path::new().to_path_buf` is heavily used within this PR, so we probably don't want to make this change in this PR. However, if you agree that this is a good change, I can open a separate PR addressing similar issues in one go. What do you think @jkmassel?

Reply:
Yeah, I'm good with any change that makes things more idiomatic for sure!!